core/input/input_event.cpp

```diff
@@ -1186,10 +1186,10 @@ static const char *_joy_button_descriptions[(size_t)JoyButton::SDL_MAX] = {
 };
 
 String InputEventJoypadButton::as_text() const {
-	String text = "Joypad Button " + itos((int64_t)button_index);
+	String text = vformat(RTR("Joypad Button %d"), (int64_t)button_index);
 
 	if (button_index > JoyButton::INVALID && button_index < JoyButton::SDL_MAX) {
-		text += vformat(" (%s)", _joy_button_descriptions[(size_t)button_index]);
+		text += vformat(" (%s)", TTRGET(_joy_button_descriptions[(size_t)button_index]));
 	}
 
 	if (pressure != 0) {
```

Comment:
Missed that one?

Reply:
Which one do you mean?

Reply:
"Pressure" on the next line.

Reply:
Updated. Didn't turn `pressure` into a `%f` because the default precision is different.

Reply:
It's a pre-existing issue but maybe we should add a space between the RTR and the value?